SPECS/hdf5/hdf5.spec

```diff
@@ -166,6 +161,14 @@ ln -s ../configure .
 %configure \
   %{configure_opts} \
   --enable-cxx \
+  --enable-hlgiftools \
+%ifarch aarch64
+  # temporarily disable _FLOAT16 for ARM64 until a fix is checked-in.
+  # See:
+  # - https://github.com/HDFGroup/hdf5/pull/4495
+  # - https://github.com/HDFGroup/hdf5/pull/4507
```

Comment:
Thanks for the comment. I vaguely remember Bash may not like a comment in the middle of a list of arguments. If the build breaks because of that, you can move the comments to right before the `%configure \` line

Reply:
yep, it broke :) I've moved it out.